### PR TITLE
NewsManager with Room persistence lib

### PR DESCRIPTION
**Pull request by psukys**
## Issue

Partly solves #560 by converting NewsManager queries into Room persistence ORM.

DAOs are basically just used queries moved to Room's lib. I *believe* that future refactoring and optimizations could be applied for queries (possibly moving more logic into NewsManager) in the future.

This might look like a lot of changes due to LOC added, but most of it is tests.

## Screenshot
TBA

## Why this is useful for all students

Queries are only in their DAOs, which simplifies interactions.

### CODE REVIEW

**Comment by kordianbruck** [comment]
@psukys work in progress right?

**Comment by psukys** [comment]
@kordianbruck correct, forgot to attach label

**Comment by psukys** [comment]
Some `NewsDao` tests are failing, will investigate *tomorrow*

**Comment by kordianbruck** [comment]
please feel free to rename `dataAccessObjects` to `dao`

**Comment by psukys** [comment]
@kordianbruck and @pfent , PR's ready to review.

**Comment by kordianbruck** [comment]
@pfent willst deinen Senf noch dazu geben?

File: app/src/main/java/de/tum/in/tumcampusapp/adapters/NewsAdapter.java [hunk 1]
```diff
@@ -94,27 +97,25 @@ public static void bindNewsView(NetUtils net, RecyclerView.ViewHolder newsViewHo
...
-        c.moveToPosition(position);
-        bindNewsView(net, holder, c);
+        bindNewsView(net, holder, news.get(position), mContext);
     }
 
     @Override
     public int getItemViewType(int position) {
-        c.moveToPosition(position);
-        return "2".equals(c.getString(1)) ? 0 : 1;
+        return "2".equals(news.get(position).getTitle()) ? 0 : 1;
```
[the remark this answers is not in the document]
Comment (kordianbruck) [reply app/src/main/java/de/tum/in/tumcampusapp/adapters/NewsAdapter.java]: Also it should not be `getTitle` in this case :S

**Pull request merged**